websites-css/aboutus.css

```diff
@@ -15,7 +15,13 @@
     justify-content: center;
     flex-direction: column;
     gap: 70px;
+<<<<<<< HEAD
+    padding: 30px 100px;
+    
```

Comment:
This is an unresolved merge conflict, you shouldn't push up code where theres merge conflicts that you haven't taken care off.

If you're not sure how to deal with merge conflicts then you should contact Rob